src/src/DataTypes/SPI_options.h

```diff
@@ -0,0 +1,18 @@
+#ifndef DATASTRUCTS_SPI_OPTIONS_H
+#define DATASTRUCTS_SPI_OPTIONS_H
+
+#include <Arduino.h>
+
+enum class SPI_Options_e {
+  None = 0,
+  Vspi,
+  Hspi,
+  UserDefined
```

Comment:
Since this is stored... maybe leave some room between the options and UserDefined.
I can imagine there may be other options on future chips.